.github/workflows/ci_checks.yml

```diff
@@ -17,9 +17,8 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-        cache: 'pip'
```

Comment:
there's something weird going on with caching where it's crashing because it's looking for the deleted `requirements.txt` file. I imagine that eventually the old caches will clear and we could add these back, but just running it once without caches didn't seem to fix the problem, strangely. in any event, installing is fast, so caching doesn't seem super important right now.